images/scripts/rhel-9-8.bootstrap

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
This is missing an image-trigger entry. I realize this is an expensive PR, so please do it as a follow-up unless you have to push again.